R/nest.R

```diff
@@ -49,21 +49,6 @@ nest.data.frame <- function(data, ..., .key = "data") {
   }
   nest_vars <- setdiff(nest_vars, group_vars)
 
-  data <- dplyr::ungroup(data)
-  if (is_empty(group_vars)) {
-    return(tibble(!! key_var := list(data)))
-  }
-
-  out <- dplyr::select(data, !!! syms(group_vars))
-
-  idx <- dplyr::group_indices(data, !!! syms(group_vars))
-  representatives <- which(!duplicated(idx))
-
-  out <- dplyr::slice(out, representatives)
-
-  out[[key_var]] <- unname(split(data[nest_vars], idx))[unique(idx)]
-
-  out
+  data <- dplyr::select(data, !!!group_vars, !!!nest_vars)
```

Comment:
Would you mind making this a condition on `packageVersion()` dplyr so I can release in the next couple of days?

Reply:
(And hence remove the remote)